hardware/README.md

```diff
@@ -9,8 +8,9 @@ This sets it to calibration mode.
 After a particular delay the signal to the ESC is set back to low which completes the calibration. 
 
 ### Step to calibrate ESC
+${\color{red}REMOVE \space PROPELLERS \space FROM \space QUADCOPTER}$
```

Comment:
Try 
```html
<p style="color:red;">REMOVE PROPELLERS FROM QUADCOPTER</p>
```

Reply:
Hmm... you're right! Then maybe use `\rm`:

```md
${\color{red}\rm REMOVE \space PROPELLERS \space FROM \space QUADCOPTER!}$
```

which will look like this:

${\color{red}\rm REMOVE \space PROPELLERS \space FROM \space QUADCOPTER!}$

Alternatively:

⚠️⚠️⚠️ **REMOVE PROPELLERS!** ⚠️⚠️⚠️

Reply:
How about now?